lista/32/32.js

```diff
@@ -0,0 +1,20 @@
+// 32) [DESAFIO] Crie um jogo onde o computador vai sortear um número entre 1 e 5 o 
+// jogador vai tentar descobrir qual foi o valor sorteado.
+
+const prompt = require('prompt-sync')({})
+
+const sortearNum = () => {
+    let randomNumber = Math.ceil(Math.random() * 5)
+    let reposta
+    do {
+    reposta = parseInt(prompt("Digite um valor entre 1~5 :"))
+    if (reposta == randomNumber){
```

Comment:
Como vc está usando o [loose equality operator](https://www.tutorialspoint.com/strict-equality-vs-loose-equality-in-javascript#:~:text=The%20loose%20equality%20operator%20'%3D%3D,as%20well%20as%20their%20types%20.) é desnecessário o parseInt da linha de cima, pois:
![image](https://user-images.githubusercontent.com/22593352/173462132-e3ff4a8b-f569-451e-8dad-91fda251bd2b.png)